client/app/mobile/modules/series/root/masterSeries/SeriesGrid.js

```diff
@@ -12,12 +13,19 @@ class SeriesGrid extends Component {
     render() {
         return(
             <FlatList 
-                data={this.props.series}
+                style={ styles.container }
+                data={ this.props.series }
                 renderItem={ ({ item }) => this.renderSerieCard(item) }
-                keyExtractor={(item) => item.id}
+                keyExtractor={ (item) => item.id }
```

Comment:
Be consistent if you are going to add spaces or not. There are other places where you don´t use blanks